src/Microsoft.Identity.Client/ApiEvent.cs

```diff
@@ -72,6 +72,7 @@ public string UserId
         public bool WasSuccessful
         {
             set => this["was_successful"] = value.ToString();
+            get => this["was_successful"] == "True";
```

Comment:
Good catch! I chose "True" only to match the default behavior in C# of `true.ToString()` gives us a "True". That being said, now I think our telemetry spec should clearly define the case, so the implementation would follow. Need @addev-ashish 's clarification on this.

On the other hand, the other day @addev-ashish mentioned we are heading toward using raw EventBase class family as the telemetry receiver API surface, then we would just use native bool. The question is when.